src/include/storage/page/b_plus_tree_internal_page.h

```diff
@@ -40,27 +40,9 @@ class BPlusTreeInternalPage : public BPlusTreePage {
 
   auto KeyAt(int index) const -> KeyType;
   void SetKeyAt(int index, const KeyType &key);
```

Comment:
It seems that the API is not paired. e.g., we have APIs to read key and set key, but we only have API to read value without setting value... Can you think of any better way of stripping this part?

Reply:
Hmm, this is one area where I feel adding a Set operation might be confusing. Especially in the leaf page where I feel that a SetKey operation does not make sense, but GetKey does.